Source/Tools/Editor/Editor.cpp

```diff
@@ -233,7 +235,6 @@ void Editor::Start()
     {
         if (GetCommandLineParser().got_subcommand(cmd->GetTypeName().c_str()))
         {
-            context_->GetSubsystem<Log>()->SetLogFormat("%v");
```

Comment:
Does this not break log forwarding to console tab?

Reply:
It didn't, but I reverted it just in case it caused issues on Linux